Exoft.Gamification.Api.Services/Services/AuthService.cs

```diff
@@ -18,26 +19,26 @@ namespace Exoft.Gamification.Api.Services
 {
     public class AuthService : IAuthService
     {
-        private readonly IUserService _userService;
-        private readonly IJwtSecret _jwtSecret;
+        private readonly UserRepository userRepository;
+        private readonly IJwtSecret jwtSecret;
         private readonly IMapper mapper;
         
-        public AuthService(IUserService userService, IJwtSecret jwtSecret, IMapper mapper)
+        public AuthService(UsersDbContext context, IJwtSecret jwtSecret, IMapper mapper)
         {
-            _userService = userService;
-            _jwtSecret = jwtSecret;
+            this.userRepository = new UserRepository(context);
```

Comment:
wow.... new UserRepository(context)???

You should inject Irepository... forget about context here.